.github/workflows/mocha.yml

```diff
@@ -45,10 +47,12 @@ jobs:
           - 14
           - 16
           - 18
-          - 19
+          - 20
```

Comment:
Might make sense to have adding Node 21 as a followup issue? https://nodejs.org/en/blog/announcements/v21-release-announce

Reply:
There are currently issues with node21 and actions/setup-node. Times for some reason out. I had first node 21 but because of that i removed it.